### ISSUE TITLE
handling status event cancelled

### ISSUE THREAD

**Issue by MellowVoidUser** [SimpleMobileTools/Simple-Calendar#1108]
Currently when I change an event on my desktop (on nextcloud) as cancelled is shows in the nextcloud-calendar-webinterface as strike-through text with different color. However on my android device using simple-calendar this info is not shown and it seems like a regular appointment. Any plans to change this behaviour would be apriciated!

**Comment by pcsaunak** [comment]
@tibbi reading this issue, it seems to be straight forward where upon cancelling an event on the server side it should be synced with client as well. Whats your take on this, I was thinking of working on it.

**Comment by tibbi** [comment]
I wasnt researching it, not sure how are events cancelled. You can take a look at it. I would just remove cancelled events for now, there is no support for cancelled events or striking through now.

**Comment by pcsaunak** [comment]
> I wasnt researching it, not sure how are events cancelled. You can take a look at it. I would just remove cancelled events for now, there is no support for cancelled events or striking through now.

Sounds good. I will look through it and try to understand how it works and the implementation would be to remove a cancelled event.

**Comment by pcsaunak** [comment]
I created an event and cancelled / deleted the event. So once the app called the "refreshCalDAVCalendars" the event was removed from the app. I believe this functionality is working fine. Thought I haven't checked if there is a background job which updates the events even if I have not opened the app. 

Another thing that I noticed is, the app takes a while to call "refreshCalDAVCalendars" which may give an impression to end users that the event hasn't been deleted / cancelled / removed from the app. However once the call is complete the event which I had deleted from the web calendar got removed from the app.

**Comment by tibbi** [comment]
make sure you are trying it with NextCloud, not Google. There can be many differences between different CalDAV services.

**Comment by pcsaunak** [comment]
> make sure you are trying it with NextCloud, not Google. There can be many differences between different CalDAV services.

Ok . never used nextcloud seems like a paid service ? Let me try it with other calendar services that I have and I guess that should be of help

**Comment by tibbi** [comment]
it isnt paid, you can try it out for like a month for free

**Comment by pcsaunak** [comment]
@tibbi I have added multiple email Ids to the gmail app but those do not come up when I try to enable caldavsync in the settings page. How can we sync calendars from associated with different email IDs.

**Comment by tibbi** [comment]
you need to have Google Calendar installed if you want to sync via Google. For NextCloud you will need some sync adapter like DAVx5

**Comment by pcsaunak** [comment]
> you need to have Google Calendar installed if you want to sync via Google. For NextCloud you will need some sync adapter like DAVx5

So I did a test using DAVx5 and a Zoho email and I saw similar behaviour, where if I create an event and force sync the app the event shows up and after deleting the event and than force sync of DAVx5 and the calendar app, the event gets removed from the calendar. 

Since I am using a one plus device I had to force sync DAVx5.

**Comment by tibbi** [comment]
try not deleting the event, just somehow setting its status to Cancelled

**Comment by rfc2822** [comment]
Cancelled events are marked by setting [STATUS](https://developer.android.com/reference/android/provider/CalendarContract.EventsColumns#STATUS)=[STATUS_CANCELED](https://developer.android.com/reference/android/provider/CalendarContract.EventsColumns#STATUS_CANCELED).

This can happen for instances of recurring events, but also for non-recurring events or the main recurring event.

Etar strikes them through, like Thunderbird does: https://github.com/Etar-Group/Etar-Calendar/issues/278

**Comment by graue70** [comment]
related to #808

**Comment by dupondje** [comment]
It would be very nice to have this implemented.
I sync my Exchange agenda with Nine Mail to my android phone.

Now sometimes 1 occurrence of a recurring meeting is cancelled, having it 'STATUS:CANCELLED' in the ics.
But unfortunately it's still shown in the calendar app.
